custom-alerts/custom-alerts.yaml

```diff
@@ -65,4 +65,12 @@ spec:
         severity: warning
       annotations:
         description: external-dns container has not been running in the namespace 'kube-system' for 5 minutes.
-        summary: external-dns is down
\ No newline at end of file
+        summary: external-dns is down
+    - alert: K8SNodeNotReady5mins
```

Comment:
I'm not a big fan of the alert name. Perhaps 'NodeNotReady' would suffice?

Reply:
There is already an Alert called K8SNodeNotReady which comes with kube-prom however that alerts after an hour of a node being set to NotReady. This alert would be more of a notification if nodes are terminating and recreating behind the scenes. 
Kept the hour one too incase nothing happens/resolves between 5 mins and hour and the node is still set to NotReady

Reply:
I see. The K8SNodeNotReady is also a warning. Can you just lower the time trigger on this alert?

Reply:
Yes possible. The idea was 2 alerts:  5 min one was to make sure we know its happened and we get notified / have a record - nothing needs to be done. The 1 hour one would be a case we had to actually go and resolve as if it hasn't resolved by then it needs human intervention. 

thoughts?